lib/imgix/rails/image_tag.rb

```diff
@@ -3,9 +3,28 @@
 class Imgix::Rails::ImageTag < Imgix::Rails::Tag
 
   def render
-    @tag_options[:srcset] = srcset
-    @tag_options[:sizes] ||= '100vw'
+    url = ix_image_url(@source, @path, @url_params)
 
-    image_tag(ix_image_url(@source, @path, @url_params), @tag_options)
+    if @lazy
+      @tag_options[:data] ||= {}
+      @tag_options[:data][:srcset] = srcset
+      @tag_options[:data][:sizes] ||= '100vw'
+      @tag_options[:data][:src] = url
```

Comment:
Also I noticed: we need to append the `lazyload` class to `@tag_options[:class]` (and any other classes the user specified) in order for this to work with [lazysizes](https://github.com/aFarkas/lazysizes#how-to).

> Add the class "lazyload" to your images/iframes

Reply:
That'd be up to the user. I don't think we should tie-in an specific library to this gem -- there are other options someone might already be using.

Reply:
To give you an example, sometimes I use lazypreload on some images.